src/celestia/qt/qtglwidget.cpp

```diff
@@ -117,6 +117,8 @@ static GLContext::GLRenderPath getBestAvailableRenderPath(const GLContext& /*glc
 
 void CelestiaGlWidget::initializeGL()
 {
+    appCore->setScreenDpi(logicalDpiY() * devicePixelRatioF());
```

Comment:
is it better to use a fixed value 96 here? on Mac `logicalDpiY()` produces 72, which still makes everything looks smaller than it should be.

Reply:
i have no idea as i don't have any highdpi devices to test